src/modules/Dial.js

```diff
@@ -10,34 +11,77 @@
 
 const Timezone = require('moment-timezone');
 
+function getElementByClassOrId(elementName, parentId, defaultElementName) {
+  if (!elementName && !parentId)
+    return null;
+
+  const elementNameAsId = elementName ? "#" + elementName : "#" + defaultElementName;
+  const elementNameAsClass = elementName ? "." + elementName : "." + defaultElementName;
+  const parentIdSelector = "#" + parentId;
+  let element;
+
+  if (parentId) {
+    element = document.querySelector(parentIdSelector + " " + elementNameAsClass) || document.querySelector(parentIdSelector + " " + elementNameAsId);
+
+    if (!element && elementName) {
+      // Log it and try to retrieve it by id in the whole document instead as last resource
+      console.warn("Element '" + elementName + "' could not be found inside '" + parentId + "'");
+      element = document.getElementById(elementName);
+    }
+  } else {
+    element = document.getElementById(elementName);
+  }
+
+  return element;
+}
+
 class Dial {
-  constructor(settings, parentWatch) {
-    this.error = false;
-    this.errorChecking(settings);
+  constructor(settings, parentWatch, parentId) {
+    try {
+      this.name = settings.name;
+      this.hands = {};
+
+      let hourHandName, minuteHandName, secondHandName;
+
+      if (settings.hands) {
+        hourHandName = settings.hands.hour;
+        minuteHandName = settings.hands.minute;
+        secondHandName = settings.hands.second;
+      }
+      this.hands.hour = getElementByClassOrId(hourHandName, parentId, "hands-hour");
+      this.hands.minute = getElementByClassOrId(minuteHandName, parentId, "hands-minute");
+      this.hands.second = getElementByClassOrId(secondHandName, parentId, "hands-second");
 
-    if (this.error) return;
+      this.retrograde = {};
 
-    this.name = settings.name;
-    this.hands = {};
-    if (settings.hands.hour)
-      this.hands.hour = document.getElementById(settings.hands.hour);
-    if (settings.hands.minute)
-      this.hands.minute = document.getElementById(settings.hands.minute);
-    if (settings.hands.second)
-      this.hands.second = document.getElementById(settings.hands.second);
+      let retrogradeSecondHand, retrogradeSecondHandName,
```

Comment:
I would prefer against creating additional variables just to likely overwrite them. I feel the `settings.retrograde.second.max || 180` is more efficient and a bit cleaner.